src/modules/common/components/customRoute/index.js

```diff
@@ -46,13 +46,22 @@ const CustomRoute = ({ path, exact, isPrivate, forbiddenTokens, component, t, hi
     );
   }
 
+  // Redirect back to actual path when an account is not reclaimable
+  if (isMigrated && path === routes.reclaim.path) {
+    history.goBack();
+    return <div/>
```

Comment:
please redirect it to the dashboard 
```suggestion
     return <Redirect to={`${routes.dashboard.path}`} />;
```

Reply:
@soroushm  I think the scenario is different here, imagine you have already reclaimed the tokens and for some reason you navigate to explicitly to `/reclaim` then you should be redirected to where you were last e.g if i was in `/validators` tab and typed `/reclaim` then i should be navigated back to `/validator` and not `/dashboard`?